monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -444,6 +454,7 @@ class ReportPageComponent extends AuthComponent {
           <h3>Machine related recommendations</h3> : null}
         <div>
           {this.generateIssues(this.state.report.recommendations.issues)}
+          {this.tunnelingIssueComponent}
```

Comment:
We don't need to store this in the state. We can just call `tunnelIssueReport` component here and pass machines and agents. If there's no tunneling issue, the component can return `<></>`